src/logcollector/read_win_event_channel.c

```diff
@@ -559,6 +571,18 @@ void win_start_event_channel(char *evt_log, char future, char *query)
         goto cleanup;
     }
 
+    if (query) {
+        /* Create copy of query string */
+        if ((channel->query = strdup(query)) == NULL) {
+            mferror(
```

Comment:
Why using `mferror` instead of `merror`. If they work similar, use `merror` like the rest of the module.

Reply:
`mferror` is used just lines above that's why I decided to used it.